sdk/logical/events.go

```diff
@@ -12,6 +12,10 @@ import (
 
 // common event metadata keys
 const (
+	// EventMetadataPath is used in event metadata to show the API path the client must have the `subscribe` capability
+	// on in order consume the event. It is recommended that the event path metadata field uses that api path that was invoked
+	// in order to generate the event
+	EventMetadataPath = "path"
```

Comment:
Should we mention in the comment that the plugin mount will be prepended to this path?

Reply:
yes! - updated the comments